bogota-july-2018/Calculator-JS/DaniloPlazas/Calculator HTML/scriptJS.js

```diff
@@ -0,0 +1,129 @@
+var num_press = 0.0; 	//Number pressed by the user
+var opt_press = ""; 	//Arithmetic Operation pressed by the user
+var num_acum = 0.0;		//Accumulated Number
+var opt_pend = ""; 		//Arithmetic Operation pending execution
+
+/*
+	FUNCTION: pres_num
+	ARGUMENTS: number (int)
+	RETURN: NA
+*/
+function pres_num(number)
+{
+	var concatenate = "";
+
+	if (num_press === 0) {
+		num_press = number;
+	}else
+	{
+		concatenate = "" + num_press + number;
+		num_press = parseFloat(concatenate);
+	}
+
+	console.log(num_press)
+
+	document.getElementById('result2').innerText = num_press;
+}
+
+/*
+	FUNCTION: pres_opt
+	ARGUMENTS: option (string)
+	RETURN: NA
+*/
+function pres_opt(option)
+{
+	switch(option)
+	{
+		case "sum":
+
+			num_acum = executeOperation(opt_pend, num_acum, num_press);
```

Comment:
In all of these cases youre doing almost the same, please extract those 5-6 lines to a function, and send it the needed params to make it work as expected.

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself